starter-code/src/app/contact-list/contact-list.component.ts

```diff
@@ -19,6 +20,9 @@ export class ContactListComponent implements OnInit {
   addContact(){
     console.log("Add contact has been called");
     // add contact to contacts list
+    this.contacts.push(this.newContact);
+    this.addTaskValue = null;
```

Comment:
Interesting choice but not needed in this case. Setting this.newContact = {} would be enough